.travis.yml

```diff
@@ -129,7 +129,7 @@ before_script:
   - #PERL5OPT="$PERL5OPT $HARNESS_PERL_SWITCHES" plackup -Ilib --port 5001 --server HTTP::Server::PSGI tools/starman.psgi >/dev/null 2>/dev/null &
   - PERL5OPT="$PERL5OPT $HARNESS_PERL_SWITCHES" plackup -Ilib --port 5001 --server HTTP::Server::PSGI tools/starman.psgi --access_log /tmp/plackup-access.log 2> /tmp/plackup-error.log &
   - sleep 5  # give starman a chance to start
```

Comment:
Since this sleep remains in place, I really, really don't like the added complexity of the solution (albeit less invasive than Yves's proposal).
I really do not think we need to optimize for this 5s sleep. Instead, we need to look at optimizing recurring sleeps or doing other stuff during those recurring  sleeps (like in the web framework).